local/install_deps_linux.bash

```diff
@@ -98,7 +98,7 @@ else
 
   export CLOUD_SDK_REPO="cloud-sdk"
   export APT_FILE=/etc/apt/sources.list.d/google-cloud-sdk.list
-  export APT_LINE="deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main"
+  export APT_LINE="deb https://packages.cloud.google.com/apt $CLOUD_SDK_REPO main"
```

Comment:
Nevermind, you've answered this question already somehow :-)

Reply:
Tested on an ubuntu20.04 Docker image of mine. About a month or so ago `http` worked but now it fails whereas `https` works

Reply:
For version `2.6.0` http seems to still work for `google-cloud-sdk` (for install_deps.bash). Trying to deploy from master was where I needed https

Reply:
I want to think about how we can do this in a way that adds less complexity, since this isn't a change that's useful to us. I think maybe through a config file.